cli/test/smokehouse/config/config.js

```diff
@@ -0,0 +1,10 @@
+/** @type {Record<string, Array<string>>} */
+const exclusions = {
+  // runner: ['excluded1', 'excluded2']
+  // e.g. 'cli': ['a11y']
+  'cli': [],
```

Comment:
can you fill out the exclusions from the GH actions .yml files?

Reply:
Updated the list of disabled tests from `devtools` runner. The comment in file and the list was in sync except for these two tests: `redirects-history-push-state` and `redirects-scripts`.

Do you happen to know why these two are disabled in devtools so I can update the comment?

Reply:
Updated to group the redirects together.